### ISSUE TITLE
Cant get it to work with Steinberg CI2+

### ISSUE THREAD

**Issue by fanatm** [Maxx53/NoCableLauncher#21]
Hi,

I'm using Steinberg ci2+ for my Steam version of rocksmith 2014.
Plug my guitar to Input 1 (which have  Hi-Z input).
I can open the game just fine but the patch doesnt seem to work.
I have tried selecting real tone or microphone in game, both didnt work
Do you have any idea what went wrong?

Thanks
![rsmith](https://cloud.githubusercontent.com/assets/25891053/23110461/2e155560-f754-11e6-85bf-2b0e8477e659.jpg)

**Comment by Maxx53** [comment]
1st: make sure, your card's output format is 48khz, 16bit, mono
2nd: try other card input!

**Comment by Maxx53** [comment]
https://github.com/Maxx53/NoCableLauncher/issues/6

**Comment by fanatm** [comment]
well, looks like my card only can do 44.1/48khz, 24bit, stereo and the format list box is greyed out
So I guess I'm running out of option here?

**Comment by Maxx53** [comment]
Can you set 48khz through your card's applet in control panel (driver settings)?

**Comment by fanatm** [comment]
yes, the yamaha driver can only choose 44.1 and 48. No other option

**Comment by fanatm** [comment]
I'm not in front of my system right now, but i think there's a mono button on the device. It suppose to split the L/R channel input. I'll check that a.s.a.p
